analyzer/emerald/incoming.go

```diff
@@ -188,7 +187,7 @@ func extractRound(sigContext signature.Context, b *block.Block, txrs []*sdkClien
 		}
 		blockTransactionData.Raw = cbor.Marshal(txr.Tx)
 		blockTransactionData.RelatedAccountAddresses = map[string]bool{}
-		tx, err := common.VerifyUtx(sigContext, &txr.Tx)
+		tx, err := common.OpenUtxNoVerify(&txr.Tx)
 		if err != nil {
 			logger.Error("error verifying tx", "tx_index", txIndex, "err", err)
 			tx = nil
```

Comment:
Is this enough, or should we also skip event processing (line 248)? Skipping events feels more consistent.

Reply:
events are an independent take on what happened in this transaction. and it's the runtime itself's word vs our different-language, different-sdk word. thus, the events list is a more reliable piece of information than us being able to decode the tx or not. so we should not skip event processing under this condition where we know our decoding has diverged from the runtime's decoding

Reply:
>  thus, the events list is a more reliable piece of information than us being able to decode the tx or not

This is also where I am with regards to `Consensus` events.

Reply:
Nice explanation, thanks! Makes sense.